#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# (c) ACE 

import os

class Config(object):
    # get a token from @BotFather
    pass
    """
    BOT_TOKEN = os.environ.get("BOT_TOKEN", "7378952429:AAGJcprLeKeKw9b9oIdPsUy8rZKzh71SJ6w")
    API_ID = int(os.environ["API_ID", 27355149]
    API_HASH = os.environ["API_HASH", "68bcb1ab22bd80bbae9d66ccddf540c2"]
    AUTH_USERS = "6401497985"""
# jisko bhi is repo ko deploy karne me error aa raha ho ( https://t.me/Stubborn1223 ) ye mere TG ka id ha message karke iska screen shot send kar dena me error solve kar dunga. 
